style/js/shield_defs.js

```diff
@@ -140,9 +140,22 @@ export function loadShields(shieldImages) {
 
   shields["US:US:Truck"] = banneredShield(shields["US:US"], ["TRK"]);
   shields["US:US:Spur"] = banneredShield(shields["US:US"], ["SPUR"]);
+  shields["US:US:Connector"] = banneredShield(shields["US:US"], ["CONN"]);
   shields["US:US:Bypass"] = banneredShield(shields["US:US"], ["BYP"]);
   shields["US:US:Business"] = banneredShield(shields["US:US"], ["BUS"]);
+  shields["US:US:Business:Alternate"] = banneredShield(shields["US:US"], [
+    "BUS",
+    "ALT",
+  ]);
+  shields["US:US:Business:Truck"] = banneredShield(shields["US:US"], [
+    "BUS",
+    "TRK",
+  ]);
   shields["US:US:Alternate"] = banneredShield(shields["US:US"], ["ALT"]);
+  shields["US:US:Alternate:Business"] = banneredShield(shields["US:US"], [
+    "ALT",
+    "BUS",
+  ]);
```

Comment:
This network value no longer exists on the map.  On review, we found this was bannered as "Business Alternate", and changed the sole route relation with this network value.

```suggestion
```